portfolio/src/main/webapp/script.js

```diff
@@ -48,17 +48,20 @@ function getCommentData() {
     const commentListElement = document.getElementById(FETCH_ID);
     comments.forEach((comment) => {
       commentListElement.appendChild(
-          createListElement(comment.name, comment.text));
+          createListElement(comment.email, comment.text, comment.imageUrl));
     })
   });
 }
 
 /**
  * Create a list element for comments to be formatted when displayed.
  */
-function createListElement(name, text) {
+function createListElement(email, text, imageUrl) {
   const liElement = document.createElement(COMMENT_CHILD_TAG);
-  liElement.innerText = name + ': ' + text;
+  const imgElement = document.createElement('img');
+  imgElement.src = imageUrl;
```

Comment:
is the comment always going to have an image? should we check that imageUrl is not empty before creating the imgElement?

Reply:
I was thinking I didn't have to because DataServlet sets the URL to null if there's no file but it makes sense to check for null and not append a broken image if it is null